data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -807,10 +789,17 @@ def build_academy_data(
     academies["Email"] = ""
     academies["HeadEmail"] = ""
 
+    academies = academies.merge(
```

Comment:
`Trust UPIN` is now pulled in from the Central Services file so the ordering of the academies/central-services/trusts* merging has changed here.

*this should probably be removed entirely when the part-year-trusts work is done.